chart/values.yaml

```diff
@@ -709,6 +709,7 @@ config:
     rbac: 'True'
   celery:
     default_queue: celery
+    worker_concurrency: 16
```

Comment:
this seems potentially misleading because, if i understand correctly, it affects only the keda query but not the actual work concurrency setting.

Reply:
It does affect the actual concurrency setting:

https://github.com/apache/airflow/blob/cddbf9c11d092422e6695d7a5a5c859fdf140753/chart/templates/configmaps/configmap.yaml#L37-L43

Reply:
oh nice